data/pc/1.19.3/protocol.json

```diff
@@ -2396,9 +2392,9 @@
         "packet_profileless_chat": [
         	"container",
           	[
-    		  {
```

Comment:
bad tabulation

Reply:
I see. Don't know how that slipped in. Will update

Reply:
Fixed.